libobs/obs-encoder.c

```diff
@@ -1465,9 +1467,14 @@ static bool buffer_audio(struct obs_encoder *encoder, struct audio_data *data)
 	size_t offset_size = 0;
 	bool success = true;
 
-	if (!encoder->start_ts && encoder->paired_encoder) {
+	struct obs_encoder *paired_encoder = NULL;
+	/* Audio encoders can only be paired to one video encoder */
+	if (encoder->paired_encoders.num)
```

Comment:
If the comment is true, shouldn't this code also check if `num` is exactly `1` and fail otherwise?

Reply:
Could be done, but `pair_encoers` will not add a video encoder here if the audio encoder already has one, so it should never be more than one. It was more meant to clarify as to why this isn't iterating over them like the video encoder counterpart is.